src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs.mdx

```diff
@@ -1,47 +1,176 @@
-
-The specifics of how logs get annotated with the `service.name` resource attribute depends on the application's environment:
-
-* Applications may produce structured JSON logs, which you can configure to include `service.name` as another field.
-* You can deploy applications alongside a dedicated [Collector agent](https://opentelemetry.io/docs/collector/getting-started/#agent) instance, which you can configure with a [Resource processor](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource) to annotate logs with the `service.name` attribute.
-
-Optionally, additional application [trace context](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/sdk_exporters) (sometimes called execution context) can be propagated to log messages. The setup and availability of this depends on the language and logging framework used by the application. The general strategy is to set up the application to write structured JSON logs and to configure it to extract trace context into specified [trace context fields](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#trace-context-fields) on available log messages. For more details, see [OpenTelemetry in the UI: Logs page](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-logs-page).
-
-The [Logs in Context with Log4j2 example in GitHub](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/java/logs-in-context-log4j2) demonstrates an end-to-end working example for a simple Java application using Log4j2.
-
-### View OpenTelemetry logs [#view-logs]
-
-Here are two ways you can view logs:
-
-* Look in our [logs UI](/docs/logs/log-management/ui-data/use-logs-ui/).
-* If your logs are correlated with an application, view them in the [context of the application](/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic#logs).
-
-### The time field [#log-time]
-
-The `timeUnixNano` field is optional according to the OpenTelemetry specification for log data. When `timeUnixNano` is not present New Relic will use the time that the data was received for the New Relic log timestamp.
+New Relic supports two primary workflows for ingesting logs generated with OpenTelemetry:
+
+* [Direct to collector](https://opentelemetry.io/docs/specs/otel/logs/#direct-to-collector): 
+    * Applications can send logs directly to the dedicated New Relic OpenTelemetry Log Protocol (OTLP) endpoint. This streamlined approach minimizes additional components.
```

Comment:
```suggestion
    * Applications can send logs directly to the New Relic OTLP endpoint.
```
OTLP != OpenTelemetry Log Protocol endpoint. "This streamlined approach minimizes additional components" repeats content on OpenTelemetry's docs. I think as a principle we need to avoid this because: 1. Less is more. 2. We're not the authority and have a history of getting it wrong. 3. The upstream doc can change and we might be left with a conflicting message.

Reply:
@jack-berg What do you mean when you say "OTLP != OpenTelemetry Log Protocol endpoint"? How was my version (New Relic OpenTelemetry Log Protocol (OTLP) endpoint) incorrect? 

And re: your other point, I added that because it might be helpful to know when/why you'd use one workflow over the other, but we can keep the content more scrubbed down if we're depending on readers going to the OTEL docs.

Reply:
OTLP is an acronym for OpenTelemetry Protocol, not OpenTelemetry Log Protocol. It was originally an acronym for OpenTelemetry Line Protocol, but "Line" was later dropped after the shorted OTLP had already caught on. OTLP is a general purpose protocol for sending logs, metrics, traces, and profiles.

> And re: your other point, I added that because it might be helpful to know when/why you'd use one workflow over the other, but we can keep the content more scrubbed down if we're depending on readers going to the OTEL docs.

Yeah I get it. Its a balance - additional words can add helpful context even if they duplicate content which exists somewhere else, but also add more surface area for mistakes and content drift from the original source. I think in this case, the principle is to focus on "what" is happening with the two workflows, not "why".